feature/experimental/bgp/otg_tests/link_bandwidth_test/README.md

```diff
@@ -112,36 +112,39 @@ bandwidth communities to routes based on a prefix match.
     * Verify expected communities are present in ATE.
     * Verify expected communities are present in DUT state.
       * Do not fail test if this path is not supported, only log results
-      * `/network-instances/network-instance/protocols/protocol/bgp/rib/afi-safis/afi-safi/ipv4-unicast/neighbors/neighbor/adj-rib-in-post/routes/route/state/ext-community-index`
-      * `/network-instances/network-instance/protocols/protocol/bgp/rib/afi-safis/afi-safi/ipv6-unicast/neighbors/neighbor/adj-rib-in-post/routes/route/state/ext-community-index`
+        * `/network-instances/network-instance/protocols/protocol/bgp/rib/afi-safis/afi-safi/ipv4-unicast/neighbors/neighbor/adj-rib-in-post/routes/route/state/ext-community-index`
+        * `/network-instances/network-instance/protocols/protocol/bgp/rib/afi-safis/afi-safi/ipv6-unicast/neighbors/neighbor/adj-rib-in-post/routes/route/state/ext-community-index`
+      * Mark test as passing if Global Administartive valuee (ASN) og link-bakdwidth extended community **send by DUT** is either `23456` or ASN of DUT. 
 
     * Expected community values for each policy
       |              | set_linkbw_0                           | not_match_100_set_linkbw_1M |
       | ------------ | -------------------------------------- | --------------------------- |
-      | prefix-set-1 | [ "link-bandwidth:100:0" ]             | [none]                      |
-      | prefix-set-2 | [  "100:100", "link-bandwidth:100:0" ] | [ "100:100" ]               |
-      | prefix-set-3 | [ "link-bandwidth:100:0" ]             | [ "link-bandwidth:100:0" ]  |
+      | prefix-set-1 | *DEPRECATED*                           | [none]                      |
+      | prefix-set-2 | *DEPRECATED*                           | [ "100:100" ]               |
+      | prefix-set-3 | *DEPRECATED*                           | [ "link-bandwidth:100:0" ]  |
```

Comment:
Hi Rafal,

 not_match_100_set_linkbw_1M expeted output shouldnt be ["link-bandwidth:23456:0" ] for 3rd prefix as we are changing  ext-community-member of ''linkbw_1M'' to 'ext-community-member = [ "link-bandwidth:23456:1M" ] ?

Same for  match_100_set_linkbw_2G   prefix3.

Reply:
Yes. changed. good catch.

Please note also new text in line 117.